rules/windows/credential_access_bruteforce_multiple_logon_failure_same_srcip.toml

```diff
@@ -0,0 +1,99 @@
+[metadata]
+creation_date = "2020/08/29"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/08/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies multiple consecutive logon failures from the same source address and within a short time interval.
+Adversaries will often brute force login attempts across multiple users with a common or known password, in an attempt
+to gain access to accounts.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Multiple Logon Failure from the same Source Address"
+note = """## Triage and analysis
+
+#### Possible investigation steps
+
+- Investigate the logon failure reason code and the targeted user names.
+- Investigate the source IP address of the failed Network Logon attempts.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Identify the source and the target computer and their roles in the IT environment.
+
+### False positive analysis
+
+- Authentication misconfiguration or obsolete credentials.
+- Service account password expired.
+- Trust relationship between the primary domain and the trusted domain issue.
+- Infrastructure or availability issue.
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- If the host is a domain controller (DC):
+  - Activate your incident response plan for total Active Directory compromise.
+  - Review the privileges assigned to users that can access the DCs, to ensure that the least privilege principle is
+  being followed and to reduce the attack surface.
+- Isolate the involved hosts to prevent further post-compromise behavior.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure all compromised accounts are
+identified. Reset passwords for these accounts and other potentially compromised credentials, such as email, business
+systems, and web services.
+- Run a full antimalware scan. This may reveal additional artifacts left in the system, persistence mechanisms, and
+malware components.
+- Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
+- Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the
+mean time to respond (MTTR).
+
+## Setup
+
+If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
```

Comment:
Can you mention the relationship of this rule with 

name = "Multiple Logon Failure Followed by Logon Success"
rule_id = "4e85dc8a-3e41-40d8-bc28-91af7ac6cf60"


Anytime there are 10+ failures followed by success, both rules will trigger